types/luxon-business-days/index.d.ts

```diff
@@ -0,0 +1,40 @@
+import { DateTime } from "luxon";
+
+export type BusinessDayNumbers = 1 | 2 | 3 | 4 | 5 | 6 | 7;
+export type BusinessDays = BusinessDayNumbers[];
+
+export type HolidayMatcher = (inst: DateTime<boolean>) => boolean;
+export interface HolidayMatchersDict {
+    [key: string]: HolidayMatcher;
+}
+
+export type HolidayHelper = (...args: any[]) => any;
+export interface HolidayHelpersDict {
+    [key: string]: HolidayHelper;
+}
+
+declare module "luxon" {
+    interface DateTime<IsValid extends boolean> {
+        holidayHelpers: HolidayHelper[] | undefined;
+        holidayMatchers: HolidayMatcher[] | undefined;
```

Comment:
```suggestion
        holidayHelpers?: HolidayHelper[] | undefined;
        holidayMatchers?: HolidayMatcher[] | undefined;
```
not sure what was the intent, but these looks like strict optional properties:
https://www.typescriptlang.org/tsconfig/#exactOptionalPropertyTypes

Reply:
Thanks for the review @peterblazejewicz 
I have updated the PR to make them optional. That actually made me catch a mistake I had made (`holidayHelpers` should've been `businessDays`)